client/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { Component } from "react";
+import "./App.css";
+import axios from "axios";
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state = {
+      projects: [],
+      actions: []
+    };
+  }
+
+  componentDidMount() {
+    axios
+      .get("http://localhost:9090/api/projects")
+      .then(res => {
+        this.setState({ projects: res.data });
+        console.log(this.state.projects);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+
+  /*     axios
+      .get("http://localhost:9090/api/actions")
+      .then(res => {
+        this.setState({ projects: res.data });
+        console.log(this.state.projects);
+      })
+      .catch(err => {
+        console.log(err);
+      }); */
+  }
+
+  addProject = newProject => {
+    axios
+      .post("http://localhost:9090/api/project", newProject)
+      .then(res => {
+        this.setState({ projects: res.data });
+      })
+      .catch(err => {
+        console.log(err);
+      });
+
+    this.props.history.push("/");
```

Comment:
I think there's probably a better way to rerender our components with updated state rather than push to the page we're currently on, but this definitely works